docs/network/smart-rollup-nodes.md

```diff
@@ -18,65 +18,91 @@ Make sure that you understand the interaction between different nodes as describ
 
 For more information about Smart Rollup nodes in general, see [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) on docs.tezos.com and [Smart Rollup Node](https://tezos.gitlab.io/shell/smart_rollup_node.html) in the Octez documentation.
 
-## Running the Smart Rollup node from a snapshot
+You can run the Etherlink Smart Rollup in many modes, but these instructions cover only the most common modes:
 
-You can start the Smart Rollup node with a snapshot of the Etherlink state to prevent it from having to compute the state from Etherlink genesis.
+- [Running the Smart Rollup node in operator mode](#running-the-smart-rollup-node-in-operator-mode)
+- [Running the Smart Rollup node in observer mode](#running-the-smart-rollup-node-in-observer-mode)
 
-For simplicity, these steps show how to run the Smart Rollup node in observer mode:
+## Running the Smart Rollup node in operator mode
+
+Running the Smart Rollup node in operator mode is the best way to participate in Etherlink because in this mode the node posts commitments about Etherlink's state to layer 1.
+These commitments validate Etherlink's state and ensure that Etherlink is processing blocks and transactions honestly according to its kernel.
+As described in [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) on docs.tezos.com, one honest node is enough to ensure that Etherlink is running correctly, but adding more nodes strengthens its security and allows users to verify for themselves that Etherlink is running as intended.
+
+Running the node in this mode requires these prerequisites:
+
+- A Tezos layer 1 node running in archive mode.
+The Smart Rollup node needs the full history from the time the snapshot was taken to the current level, which usually means connecting to an archive node, but sometimes it can work with a rolling node that keeps enough history.
+After the node has started, you can switch to a rolling node.
+You can use a public archive node for this purpose, but for better performance, switch to your own node after the Smart Rollup node has started.
```

Comment:
As mentioned on Slack, I think we need to be more strict here.

Running a SR node in operator mode should _never_ be done against a public endpoint that you don’t own. It’s not even related to performances, it’s for security purposes.

A malicious public node can potentially expose you a malicious branch, and make you compute invalid commitments.

Reply:
I really think we should explicitly say “no public endpoint when running in operator mode” because it can be harmful otherwise, and if it means having to work on the PR a bit longer, I think it’s worth it.

Maybe it’s being too defensive here, but it’s for the security of operators’ assets.

Reply:
Got it, I've added a stronger warning.